NOTICES

```diff
@@ -2156,4 +1646,4 @@ freely, subject to the following restrictions:
 2. Altered source versions must be plainly marked as such, and must not be
    misrepresented as being the original source code.
 
-3. This notice may not be removed or altered from any source distribution.
\ No newline at end of file
+3. This notice may not be removed or altered from any source distribution.
```

Comment:
Although this is probably a good change, this change is not related to the functional change. To make it easier to cherry-pick without conflict, this should be in a separate commit.

Reply:
Do you mean the newline?

Reply:
Yes